### DISCUSSION TITLE
object detection example

### DISCUSSION THREAD

**Discussion by wpfhtl** [bitcraze/discussions#873] [Q&A]
May I ask whether there is any example to implement object detection using AI deck?

**Comment by krichardsson** [comment]
You can find examples for the AI-deck in the [aideck-gap8-examples](https://github.com/bitcraze/aideck-gap8-examples) repository. Perhaps the [face detection example](https://github.com/bitcraze/aideck-gap8-examples/tree/master/examples/image_processing/FaceDetection) is what you are looking for?

**Reply by Brittanyzhou** [reply]
Hi I want to do object detection, like detect cups, books something like this. I try to use other object detection methods, but can't produce lib files, I want to know why.

**Reply by krichardsson** [reply]
@Brittanyzhou This sounds like a different problem, please don't reuse discussions, see #55.
You might find your answer in #1018
